src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -401,7 +420,7 @@ BoundExpression visitArgumentsAndFinishRewrite(BoundCall node, BoundExpression?
                     ref temps,
                     invokedAsExtensionMethod);
 
-                InterceptCallAndAdjustArguments(ref method, ref rewrittenReceiver, ref rewrittenArguments, ref argRefKindsOpt, invokedAsExtensionMethod, node.InterceptableNameSyntax);
+                InterceptCallAndAdjustArguments(ref method, ref rewrittenReceiver, ref rewrittenArguments, ref argRefKindsOpt, ref temps, invokedAsExtensionMethod, node.InterceptableNameSyntax);
```

Comment:
>InterceptCallAndAdjustArguments [](http://example.com/codeflow?start=16&length=31)

Is implementation of ```Instrumenter.InterceptCallAndAdjustArguments``` called below subject to the same issue?

Reply:
Based on the instrumenters which currently override this method in the compiler, I think it is not subject to #71657 because it will not substitute a call to instance method with a call to static method.